mount/temp.go

```diff
@@ -59,6 +59,22 @@ func WithTempMount(ctx context.Context, mounts []Mount, f func(root string) erro
 			}
 		}
 	}()
+
+	// The volatile option of overlayfs doesn't allow to mount again using the
```

Comment:
I would suggest adding a TODO to add checking when the kernel adds support for reusing mounts.

Reply:
Or put it in your commit message.

Reply:
Makes sense to me. Added a comment.